apps/dot_ext/utils.py

```diff
@@ -136,7 +138,7 @@ def validate_app_is_active(request):
             description=settings.APPLICATION_TEMPORARILY_INACTIVE.format(app.name)
         )
 
-    return app
```

Comment:
It currently looks like this always will return `None` for `dag`, was that the intention?

Reply:
If app exists, it gets set below on l121.

Reply:
Ah okay, missed that, thanks!